.github/styles/Bokeh/WordList.yml

```diff
@@ -0,0 +1,41 @@
+extends: substitution
+message: "Use '%s' instead of '%s' (if possible)."
+level: error
+ignorecase: false
+action:
+  name: replace
+swap:
+  '(?:API Console|dev|developer) key': API key
+  '(?:e-mail|Email|E-mail)': email
+  '(?:kill|terminate|abort)': stop|exit|cancel|end
+  '(?:ok|Okay)': OK|okay
+  '(?:WiFi|wifi)': Wi-Fi
+  '3\-D': 3D
+  'file( |-)name': filename
+  'Javascript|Java Script': JavaScript
+  'tap (?:&|and) hold': touch & hold
+  admin: administrator
+  Ajax: AJAX
+  approx\.: approximately
+  autoupdate: automatically update
+  check(-| )box: checkbox
+  data are: data is
+  fewer data: less data
+  HTTPs: HTTPS
+  ingest: import|load
+  Jupyter Notebook: Jupyter notebook
+  open-source: open source
+  Pandas: pandas (lowercase)
+  python: Python
+  regex: regular expression
+  repo: repository
+  repos: repositories
+  SHA1: SHA-1|HAS-SHA1
```

Comment:
What does "HAS" stand for?

Reply:
Good question - I don't think this really exists. This is also based on Vale's version of the Goolge developer documentation style guide' wordlist entry for "SHA", but it really does not make much sense, so I'll take this line out as well.